src/wallet.ts

```diff
@@ -316,16 +316,20 @@ export class Wallet {
 
     const { data: accessKey } = await this.viewAccessKey(accountId, publicKey)
 
-    const allowance = utils.format.formatNearAmount(
+    let allowance = utils.format.formatNearAmount(
       accessKey.permission.FunctionCall.allowance
     )
 
+    if (typeof allowance == 'undefined') {
+      allowance = '0.25'
```

Comment:
would add this to a const (if it doesn't exist already)

Reply:
done